src/control/run_go_tests.sh

```diff
@@ -44,10 +44,16 @@ function setup_environment()
 
 	source "${build_source}"
 
-	LD_LIBRARY_PATH="${SL_PREFIX}/lib:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
-	export LD_LIBRARY_PATH
-	export CGO_LDFLAGS="-L${SL_SPDK_PREFIX}/lib -L${SL_PREFIX}/lib"
-	export CGO_CFLAGS="-I${SL_SPDK_PREFIX}/include"
+	# ugh, appease the linter...
+	LD_LIBRARY_PATH=${SL_PREFIX}/lib
+	LD_LIBRARY_PATH+=":${SL_SPDK_PREFIX}/lib"
+	LD_LIBRARY_PATH+=":${SL_HWLOC_PREFIX}/lib"
```

Comment:
It's your intention to overwrite and not just extend `LD_LIBRARY_PATH` here?

Reply:
For tests, I'd prefer to, yes. Unless someone can think of a good idea not to. My thought was that it ensures that the tests either happen with an environment completely supplied from the outside, or else the test runner handles it completely. I'd kind of like to disallow external environment altogether but figured that might make life harder for local development.